CRM/MembershipExtras/Queue/Builder/Base.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+abstract class CRM_MembershipExtras_Queue_Builder_Base {
+
+  protected $queue;
+
+  protected $records = [];
+
+  protected $recordsLimit;
+
+  protected $taskCallback = [];
+
+  /**
+   * CRM_MembershipExtras_Queue_Build_BaseBuilder constructor.
+   *
+   * @param CRM_Queue_Queue $queue
+   */
+  public function __construct($queue) {
+    $this->queue = $queue;
+    $this->recordsLimit = 10;
```

Comment:
can you make this class constant, then you can also use it in tests instead of relying on knowing what the value is supposed to be.